src/Uno.UI/UI/Xaml/Controls/Image/Image.iOS.cs

```diff
@@ -287,18 +287,12 @@ public override void LayoutSubviews()
 			UpdateContentMode(newValue);
 		}
 
-		private CGSize _previousSize;
-
-		public override CGSize SizeThatFits(CGSize size)
-		{
-			size = _layouter.Measure(size.ToFoundationSize());
-
-			return _previousSize = size;
-		}
+		public override CGSize SizeThatFits(CGSize size) => _layouter.Measure(size.ToFoundationSize());
```

Comment:
You don't invoke `base.SizeThatFits` ... it might be legit, but...

Reply:
It is.